src/compiler/types.ts

```diff
@@ -4354,6 +4354,7 @@ namespace ts {
 
         /* @internal */ getLocalTypeParametersOfClassOrInterfaceOrTypeAlias(symbol: Symbol): readonly TypeParameter[] | undefined;
         /* @internal */ isDeclarationVisible(node: Declaration | AnyImportSyntax): boolean;
+        /* @internal */ isPropertyAccessible(node: Node, isSuper: boolean, writing: boolean, type: Type, prop: Symbol): boolean;

```

Comment:
I'm guessing this should be internal, unless we add some code here that determines what value `isSuper` and `writing` should be, and possibly get the type or the property symbol ourselves instead of as arguments to avoid incorrect usages of the function.

Reply:
I would definitely start this as internal. When are `isSuper` and `writing` true? I think calculating them for the caller is a good idea.

I had another idea in a comment in completions.ts

Reply:
Decided this function shouldn't (and actually can't) calculate `isSuper` and `isWrite`. This function does *not* assume we have an existing property access-y node, which we would need to calculate those arguments, so the caller should specify what are the appropriate arguments.